app/assets/stylesheets/components/_card.scss

```diff
@@ -62,17 +61,25 @@
 }
 
 .work-card-image {
-  max-height: 250px;
-  max-width: 250px;
+  max-height: 280px;
+  max-width: 280px;
   margin: 8px 0;
 }
 
 .video {
```

Comment:
Lil' thing : .video has se same margin as .work-card-image, maybe more readable to do :  

`.video, .work-card-image{
  margin: 8px 0;
}`

If we change something in the future, we don't need to repeat